packages/coinstac-ui/app/render/components/computation-field-covariates.js

```diff
@@ -1,144 +1,159 @@
-import React, { Component, PropTypes } from 'react';
+import React, { PropTypes } from 'react';
+import { Field } from 'redux-form';
 import {
+  Alert,
   Button,
   ControlLabel,
   FormControl,
   FormGroup,
   HelpBlock,
 } from 'react-bootstrap';
 
-export default class ComputationFieldCovariates extends Component {
-  constructor(props) {
-    super(props);
-
-    this.handleNewClick = this.handleNewClick.bind(this);
-    this.getHandleValueChange = this.getHandleValueChange.bind(this);
-  }
-
-  getHandleValueChange(property, index) {
-    return event => {
-      // `property` will be `null` on delete clicks
-      return this.props.onChange(
-        property ? { [property]: event.target.value } : null,
-        index
-      );
-    };
-  }
-
-  handleNewClick() {
-    this.props.onChange(
-      {
-        name: '',
-        type: null,
-      },
-      this.props.value.length
-    );
-  }
+function covariateName({
```

Comment:
Couple additional components in this module which should be moved to their own files. We can tackle when we have time to refactor.